README.md

```diff
@@ -301,6 +302,34 @@ fga models **get**
 }
 ```
 
+##### Validate an Authorization Model
+
+###### Command
+fga models **validate**
```

Comment:
once https://github.com/openfga/cli/pull/56 is merged in, this should be `fga model **validate**`